scripts/l10n/generate_po_files.py

```diff
@@ -13,31 +13,37 @@
 
 import importlib
 import inspect
-import os
-import subprocess
 import sys
 from pathlib import Path
 
+from lingua.extract import main as pot_create
+from polib import pofile
+
 
 class POGenerator:
     """Generates .po files for supported country/market entities."""
 
+    def po_merge(self, po_file: str, pot_file: str) -> None:
+        """Merge .po file with .pot"""
+        po = pofile(po_file)
+        pot = pofile(pot_file)
+        po.merge(pot)
+        po.save(po_file)
+
     def process_countries(self):
         """Processes entities in specified directory."""
         country_code_info_mapping = {}
-        module_name_path_mapping = {
-            str(path).split(os.sep)[-1].replace(".py", ""): str(path)
-            for path in Path(os.path.join("holidays", "countries")).glob(
-                "*.py"
-            )
-            if not str(path).endswith("__init__.py")
-        }
+        module_name_path_list = (
```

Comment:
Would just `modules` be okay here?
```suggestion
        modules = (
```